examples/host_filter_dag.json

```diff
@@ -38,6 +38,14 @@
       "additional_attributes": { "truncate_fragments_to": 75000000
                                 ,"output_gene_file": "reads_per_gene.star.tab" }
     },
+    {
+      "in": ["star_out"],
+      "out": "insert_size_metrics_out",
```

Comment:
This naming convention is taken from the command option `CollectInsertSizeMetrics`. I believe DNA would be a misnomer as RNA can also be used here.

Reply:
I believe insert size is suitable for this. Since we are using the PicardTools CollectInsertSizeMetrics tool, continuing to use that naming convention makes sense to me. Good to be explicit though!

Reply:
Thanks Katrina.

> DNA would be a misnomer as RNA can also be used here

I believe Charles's comment indicates that *only* DNA can be used here so my original point still stands.

Reply:
Charles' comment correctly indicates that for this iteration, only DNA makes sense. However, we have an upcoming JIRA task to add RNA support, at which point `insert_size_metrics_out` would make sense.

Reply:
Hey @gregdingle, while you were away, we chatted more offline -- we will work on adapting the step with additional calculations so it can also work for RNA in a follow-up PR. So my comment about only using DNA is out-of-date. Thank you for the review though!

Reply:
Ok sounds good. Thanks for clarifying!